packages/xeus/src/worker.ts

```diff
@@ -105,9 +105,11 @@ export abstract class EmpackedXeusRemoteKernel extends XeusRemoteKernelBase {
         name: pkg.name,
         version: pkg.version,
         repo_url: pkg.channel ? pkg.channel : '',
-        url: pkg.url ? pkg.url : '',
+        url: pkg.url ? pkg.url : pkg.channel,
```

Comment:
`pkg.channel` will never be a valid package URL unfortunately, so we can't really apply this change

```suggestion
        url: pkg.url ? pkg.url : '',
```